File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.cpp
#include <string>
#include <iostream>
#include <algorithm>
using namespace std;

bool solution(string s)
{
    bool answer = true;
    int p = 0, y = 0;
    
    for(int i=0; i<s.length(); i++){
        if(s[i] == 'p' || s[i] == 'P') p++;
        if(s[i] == 'y' || s[i] == 'Y') y++;
    }
    
   if(p!=y) answer = false;
    // [실행] 버튼을 누르면 출력 값을 볼 수 있습니다.
    cout << "Hello Cpp" << endl;

    return answer;
}

File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.js
function solution(s){
    var answer = true;

    // [실행] 버튼을 누르면 출력 값을 볼 수 있습니다.
    console.log('Hello Javascript')
    let pCount = 0, yCount = 0;
    for(let i=0; i<s.length; i++){
        if(s[i] === 'p' || s[i] === 'P') pCount ++;
        else if(s[i] === 'y' || s[i] === 'Y') yCount++;
    }
    if(pCount !== yCount) return false;

    return answer;
}